docs/application/native/guides/app-management/ui-app/dali-basic-app.md

```diff
@@ -54,7 +54,7 @@ The following table lists the callbacks for the basic signals provided by the `D
 | `BatteryLowSignal()`      | Called when the battery state is low. You can use the signal to save any crucial information in case the battery state becomes so low that the application is forcefully terminated. |
```

Comment:
On line 51, `[application control](app-controls-n.md)` link is broken. It looks like `[application control](../app-controls.md)` is correct.

Reply:
done